### PR TITLE
Validation

### PR DESCRIPTION
**Pull request by Harshtroo**

### CODE REVIEW

**Comment by Harshtroo** [comment]
image field add

**Review by Harshtroo — COMMENTED** [review]
add image field

File: book/book_crud/static/js/create_book.js [hunk 1]
```diff
@@ -26,14 +26,16 @@ $("#my-create-book").on("submit",function(event){
...
 
             for (var i=0;i< data.length;i++){
              output += `
                 <option value=${data[i].id}>${data[i].name}</option>
-             `
+                `
             document.getElementById("author_book").innerHTML = output
             }
+            console.log("output====================",output)
         }
```
Comment (Dhruvi-Makwana) [inline book/book_crud/static/js/create_book.js:29]: remove console.log

File: book/book_crud/views.py [hunk 2]
```diff
@@ -27,7 +27,10 @@ def get_create_author(request):
 
 def get_update_data(request,pk):
     book = Book.objects.get(id=pk)
-    return render(request,"book_edit.html",{"book":book})
+    
+    authors = Author.objects.all()
+    print("authors========",authors)
+    return render(request,"book_edit.html",{"book":book,"authors":authors})
```
Comment (Dhruvi-Makwana) [inline book/book_crud/views.py:31]: remove print

File: book/book_crud/views.py [hunk 3]
```diff
@@ -75,6 +78,7 @@ def book_update(request, pk):
         return Response({"error": "Book not found"}, status=status.HTTP_404_NOT_FOUND)
 
     serializer = BookSerializer(book, data=request.data)
+    print("serializer=====",serializer)
     if serializer.is_valid():
         serializer.save()
         return Response(serializer.data)
```
Comment (Dhruvi-Makwana) [inline book/book_crud/views.py:76]: try to send response with status code

**Pull request merged**